source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -533,12 +492,133 @@ namespace AdaptiveCards::Rendering::Uwp
                         m_copyStreamOperations.push_back(copyStreamOperation);
 
                         copyStreamOperation.Completed(
-                            [randomAccessStream](winrt::IAsyncOperationWithProgress<uint64_t, uint64_t> const& /*operation*/,
-                                                 winrt::AsyncStatus /*status*/) { randomAccessStream.Seek(0); });
+                            [randomAccessStream, weakThis, uiElement, parentElement, imageContainer, isAutoSize, isVisible, isImageSvg, image, stretch](
+                                auto const& /*operation*/, auto status)
```

Comment:
>) [](http://example.com/codeflow?start=70&length=1)

What is the exception/error story for these callbacks?  There is no way to propagate errors to the containing code so they are probably dropped on the floor.  I'm not sure about this specific completion handler but there is a decent chance it will treat any exceptions that leak as a failfast.

Reply:
@beervoley do you have any insights here?

We typically log errors through the renderContext, but like you said, we don't have access within the callbacks.

Reply:
Anna, can you test what happens when this callback fails? Meaning, wrong URL (httpclient cannot resolve it) or it can but it's not truly an image and the stream is just garbage info?

Reply:
If the URL cannot be resolved, the status is `Error` and the image source is not set.

If the info is garbage, the errors will be handled by `HandleAccessStreamForImageSource`